includes/admin/add-ons.php

```diff
@@ -29,16 +29,12 @@ function edd_add_ons_page() {
 
 	ob_start(); ?>
 	<div class="wrap" id="edd-add-ons">
-		<h1 class="wp-heading-inline"><?php echo edd_get_label_plural(); ?></h1>
-		<a href="<?php echo admin_url( 'post-new.php?post_type=download' ); ?>" class="page-title-action">Add New</a>
-		<hr class="wp-header-end">
-		<?php edd_display_product_tabs(); ?>
-		<h2>
+		<h1>
 			<?php _e( 'Apps and Integrations for Easy Digital Downloads', 'easy-digital-downloads' ); ?>
 			<span>
 				&nbsp;&nbsp;<a href="https://easydigitaldownloads.com/downloads/?utm_source=plugin-addons-page&utm_medium=plugin&utm_campaign=<?php echo $campaign; ?>&utm_content=All%20Extensions" class="button-primary" target="_blank"><?php _e( 'Browse All Integrations', 'easy-digital-downloads' ); ?></a>
 			</span>
-		</h2>
+		</h1>
```

Comment:
I don't love having the button inside of the `h1` but if it's not, then any admin notices will separate it from the heading, not sure how you might feel about that:

![image](https://user-images.githubusercontent.com/4853066/135334951-d98bf619-1401-44a8-8f6d-bd793b937e0b.png)

That's with changing the markup to be more consistent with other WP heading/action:
```php
		<h1 class="wp-heading-inline"><?php esc_html_e( 'Apps and Integrations for Easy Digital Downloads', 'easy-digital-downloads' ); ?></h1>
		<a href="https://easydigitaldownloads.com/downloads/?utm_source=plugin-addons-page&utm_medium=plugin&utm_campaign=<?php echo $campaign; ?>&utm_content=All%20Extensions" class="page-title-action button-primary" target="_blank"><?php esc_html_e( 'Browse All Integrations', 'easy-digital-downloads' ); ?></a>
```

Without the notices, it looks like you would want:
![image](https://user-images.githubusercontent.com/4853066/135335107-2c9d1cea-3ab8-4771-a1f5-ef374ad69d1a.png)

Reply:
this page is going to get a complete re-write eventually so let's just ignore that for now.